exchanges/kraken/kraken.go

```diff
@@ -80,16 +77,11 @@ func (k *Kraken) SeedAssets(ctx context.Context) error {
 // GetAssets returns a full asset list
 func (k *Kraken) GetAssets(ctx context.Context) (map[string]*Asset, error) {
 	path := fmt.Sprintf("/%s/public/%s", krakenAPIVersion, krakenAssets)
-
-	var response struct {
-		Error  []string          `json:"error"`
-		Result map[string]*Asset `json:"result"`
-	}
-
-	if err := k.SendHTTPRequest(ctx, exchange.RestSpot, path, &response); err != nil {
-		return response.Result, err
+	var result map[string]*Asset
```

Comment:
```suggestion
	var result map[string]Asset
```

And change return type